jobs/integr8ly/pds-install.yaml

```diff
@@ -168,9 +168,12 @@
                         dir('installation/evals') {
                             
                             String ansibleParams = ''
-                            if(GH_CLIENT_ID && GH_CLIENT_SECRET) {
+                            if (GH_CLIENT_ID && GH_CLIENT_SECRET) {
                                 ansibleParams = "-e github_client_id=${GH_CLIENT_ID} -e github_client_secret=${GH_CLIENT_SECRET}"
                             }
+                            if (ADDITIONAL_ANSIBLE_PARAMS.indexOf('eval_seed_users_count') == -1) {
+                                ansibleParams = ansibleParams + " eval_seed_users_count=15"
```

Comment:
I think `-e ` is missing, something like `ansibleParams + " -e eval_seed_users_count=15"`

Reply:
Another occurrence that needs to be fixed is also in the poc-install.yaml, just not to forget.

Reply:
@trepel Good catch. Thanks. Updated